tools/scripts/server/startServer.bat

```diff
@@ -1,30 +1,36 @@
 :: This script starts the local arma dev-server
-:: All meta-information are set via the setServerData.bat script that
+:: All meta-information are set via the setMetaData.bat script that
 :: is being called from this one
 @echo off
 
 :: set meta infos
-call %~dp0.\setMetaData.bat
+call "%~dp0.\setMetaData.bat"
+
+:: convert relative to absolute path of clib
+:: so armaserver can read it properly
+set "clib=%~dp0..\..\..\PBOs\dev\@CLib"
+pushd "%clib%"
+set "clib=%CD%"
+popd
+
+:: convert relative to absolute path of opt
+:: so armaserver can read it properly
+set "opt=%~dp0..\..\..\PBOs\dev\@OPT"
+pushd "%opt%"
+set "opt=%CD%"
+popd
 
 :: change directory into the Arma directory (in which the server-exe resides)
 cd /d "%armaDir%"
 
-set "clib=%modDir%\@CLib"
-set "opt=%modDir%\@OPT"
-
 :: Copy the server-config into the arma-dir as Arma can only read configs relative
 :: to the exe
-echo Trying to copy config file. This might take a while...
-:copyLoop
-	copy "%~dp0.\.\serverConfig.cfg" "%armaDir%" > NUL 2>&1
-
-	if not [%errorlevel%] == [0]  (
-		:: sleep 100ms
-		pathping localhost -n -q 1 -p 100 >nul
-		goto :copyLoop
-	)
+copy /Y "%~dp0.\serverConfig.cfg" "%armaDir%" > NUL 2>&1

```

Comment:
This should never fail. If so, abort! No need to retry on and on...

Reply:
It fails when you are restarting the server because windows keeps the file locked some time even after the server-instance has been killed. And in these cases the copying fails